offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectView.groovy

```diff
@@ -320,6 +320,7 @@ class CreateProjectView extends VerticalLayout{
     }
 
     private void resetInputs() {
+        String previousSpaceName = new String(resultingSpaceName.getValue())
```

Comment:
Isn't it sufficient to just remove the this.resultingSpaceName.clear() call in line 328 to keep the projectspace value in the textfield? 
```suggestion
```